pkg/hive/cell/config.go

```diff
@@ -165,3 +166,44 @@ func stringToCIDRHookFunc(from reflect.Type, to reflect.Type, data interface{})
 	}
 	return cidr.ParseCIDR(s)
 }
+
+// stringToSliceHookFunc converts strings to []string by splitting by spaces or commas.
+func stringToSliceHookFunc(from reflect.Type, to reflect.Type, data interface{}) (interface{}, error) {
+	if from.Kind() != reflect.String || to.Kind() != reflect.Slice {
+		return data, nil
+	}
+
+	raw := data.(string)
+	if raw == "" {
+		return []string{}, nil
+	}
+
+	// Split by commas first
+	if ss := strings.Split(raw, ","); len(ss) > 1 {
+		return ss, nil
+	}
+
+	// Splitting by commas didn't result in multiple values
+	// so try spaces next.
+	return strings.Split(raw, " "), nil
```

Comment:
Hmm, should I use `strings.Fields` here instead to allow multiple whitespaces as separator?

Reply:
Makes sense to me, that would be consistent with what viper does.

Reply:
Changed.